### PR TITLE
Milestone - 3 merge from dev to main

### PR DESCRIPTION
**Pull request by gauthampughazhendhi**
Please don't merge the PR yet!

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -19,6 +19,17 @@ jupyter = "^1.0.0"
...
+
+[tool.semantic_release]
+version_variable = "pyproject.toml:version" # version location
+branch = "main"                             # branch to make releases of
+changelog_file = "CHANGELOG.md"             # changelog file
+build_command = "pip install poetry && poetry build"  # build dists
+dist_path = "dist/"                         # where to put dists
+upload_to_pypi = false                      # don't auto-upload to PyPI
+remove_dist = false                         # don't remove dists
+patch_without_tag = "true"                  # always bump version on CD, even without angular commit (default is patch)
```
Comment (BooleanJulien) [inline pyproject.toml:33]: are these true/false values supposed to be 
```True``` 
and 
```False```
You've likely tested and it worked but I thought that Python was finicky about these things

**Review by BooleanJulien — APPROVED** [review]
looks good

**Review by zackt113 — APPROVED** [review]
Looks good to me!

File: pyproject.toml (hunk 1 above)
Comment (gauthampughazhendhi) [reply pyproject.toml:33]: No issues! Thanks for letting me know

**Pull request merged**